wpilibc/athena/include/SpeedControllerGroup.h

```diff
@@ -0,0 +1,38 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) FIRST 2016-2017. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#pragma once
+
+#include <functional>
+#include <vector>
+
+#include "SpeedController.h"
+
+namespace frc {
+
+class SpeedControllerGroup : public SpeedController {
```

Comment:
I don't see a point. If the user wants full control over the group of controllers like that, they can just keep a std::array or something.